docs/global-vs-local-resource-cache.md

```diff
@@ -0,0 +1,17 @@
+# Global vs local resource-cache
```

Comment:
delete

Reply:
Will remove later when I documented this in the wiki